BankManagerConsoleApp/BankManagerConsoleApp/Model/Customer.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Customer.swift
+//  BankManagerConsoleApp
+//
+//  Created by JINHONG AN on 2021/07/30.
+//
+
+import Foundation
+
+struct Customer {
+    private let _desiredTask: BankingTask
```

Comment:
get은 internal로 , set은 private로 설정해주고 싶으시다면 아래와 같은 형태로 쓴다면 많은 코드들이 줄어들 것 같아요 :)
get, set의 접근제어자를 따로 설정하지못했을때는 이런 스타일의 코드를 많이 짯지만 요즘에는 아래와같이 쓸 수있다는거 ~🍯
```suggestion
    private(set) var _desiredTask: BankingTask
```

Reply:
private(set) 을 붙이고 싶었으나 기본적으로 read-only 프로퍼티였습니다. 그래서 아래처럼 작성이 됐었는데요,
```swift
struct Customer {
    let _desiredTask: BankingTask
    let _waitingNumber: UInt
    //...
}
```
외부에서 프로퍼티를 변경할 수 없다는 점을 더 명확하게 전달하기 위해 PR에서는 get을 따로 사용했었습니다.
위 코드처럼 따로 접근제어자를 명시하지 않는 방법이 더 나을까요?